components/script/timers.rs

```diff
@@ -43,6 +43,22 @@ use crate::script_runtime::{CanGc, IntroductionType};
 use crate::script_thread::ScriptThread;
 use crate::task_source::SendableTaskSource;
 
+type TimerKey = i32;
+type RunStepsDeadline = Instant;
+type CompletionStep = Box<dyn FnOnce(&GlobalScope, CanGc) + 'static>;
+
+// OrderingIdentifier per spec ("orderingIdentifier")
+type OrderingIdentifier = DOMString;
+
+// Per-ordering queue entry: (deadline_ms, sequence, handle)
+type OrderingEntry = (u64, u64, OneshotTimerHandle);
```

Comment:
Nit: make this a struct with named fields, so it is easier to distinguish the two u64

Reply:
Good point